Library/Homebrew/unpack_strategy.rb

```diff
@@ -67,6 +67,30 @@ def extract(to: nil, basename: nil)
     unpack_dir.mkpath
     extract_to_dir(unpack_dir, basename: basename)
   end
+
+  def extract_nestedly(to: nil, basename: nil)
```

Comment:
Remind me: why is `to: nil, basename: nil` better than `to:, basename:`?

Reply:
With the latter, the argument are mandatory, with the former they are optional.

Reply:
Gotcha, thanks.